models/pathology_tumor_detection/docs/README.md

```diff
@@ -54,9 +67,9 @@ Inference is performed on WSI in a sliding window manner with specified stride.
 ## Performance
```

Comment:
You could move the determinism piece above this line: 

"### Note on determinism

By default this bundle use a deterministic approach to make the results reproducible. However, it comes at a cost of performance loss. Thus if you do not care about reproducibility, you can have a performance gain by replacing `"$monai.utils.set_determinism"` line with `"$setattr(torch.backends.cudnn, 'benchmark', True)"` in initialize section of training configuration (`configs/train.json` and `configs/multi_gpu_train.json` for single GPU and multi-GPU training respectively)."

Reply:
Done!